frontend/pages/queries/ManageQueriesPage/components/RemoveQueryModal/_styles.scss

```diff
@@ -0,0 +1,13 @@
+.remove-query-modal {
+  font-size: $x-small;
+
+  &__btn-wrap {
+    display: flex;
+    flex-direction: row-reverse;
```

Comment:
```suggestion
    justify-content: flex-end;
```
Small style nit so that the "Delete" button is to the right of the "Cancel" button.

Reply:
Done!

<img width="701" alt="Screen Shot 2021-08-03 at 10 20 05 AM" src="https://user-images.githubusercontent.com/73313222/128041666-6909c40f-9300-4dce-b968-a90879be65c1.png">